language/ui-patterns/01-documentation/animation/index.md

```diff
@@ -47,24 +64,24 @@ When animating only a single property, follow the guidelines below.
 }
 ``` 
 
-**Timing Function**
+**Timing Functions**
 
-Use bounce eases to add the snap-in factor that is typically achieved with multiple properties. Bounce eases are snappy bounces, not physics-based bounces.
+Use snap-in eases to add very strong eases to quick movements. This enhances the metaphor of elegant machine motion.
 
   * Use of Timing Functions:
-    * **Entrances** - `map-get($timing-functions, bounce-in)` (Fast to Slow) 
-    * **Exits** - `map-get($timing-functions, bounce-out)` (Slow to Fast)
-    * **Interactive Elements** - `map-get($timing-functions, bounce-in-out)`
+    * **Entrances** - `map-get($timing-functions, snap-in)` (Fast to Slow) 
+    * **Exits** - `map-get($timing-functions, snap-out)` (Slow to Fast)
+    * **Interactive Elements** - `map-get($timing-functions, snap-in-out)`
 
 ### Multiple Property Animations
 
-Multiple property animations are animations where multiple properties are being animated together. With multiple property animations there are two different options. The first option is to start one property alone, then animate any additional properties. The second option is for both properties to start at the same time, then have one property end before the other. We recommend not starting and stopping multiple properties at the same time; choose one option or the other.
+Multiple properties are animated together is known as a multiple property animation. With multiple property animations, there are two different options: to start one property alone then animate any additional properties, or to start both properties at the same time then have one property end before the other. We recommend not starting and stopping multiple properties at the same time; choose one option or the other.
```

Comment:
When more than one property is being animated, the animation is known as a multiple property animation.